packages/@react-native-windows/cli/src/index.ts

```diff
@@ -6,6 +6,19 @@
 
 import * as fs from 'fs';
 import * as path from 'path';
+import * as findUp from 'find-up';
```

Comment:
This needs to be added as a dependency to `package.json`. In our workspace, it will pull in the version used by react-native-windows-init, but would have non-deterministic behavior on the published version otherwise.